getting-started/install-timescaledb/index.md

```diff
@@ -1,12 +1,15 @@
 # Install TimescaleDB
 
-You can sign up to use hosted TimescaleDB in the cloud, managed by the team at 
-Timescale. Plans start at USD $24. Or you can elect to download TimescaleDB and 
+You can sign up to use hosted TimescaleDB in the cloud, managed by the team at
+Timescale. Plans start at USD $24. Or you can elect to download TimescaleDB and
 run it on-premises or in your own cloud infrastructure.
 
  * [Try hosted TimescaleDB][signup] for free
  * [Download and install][self-hosted] TimescaleDB in a self-hosted environment
 
+[signup]: https://www.timescale.com/timescale-signup
+[self-hosted]: /how-to-guides/install-timescaledb/self-hosted/
 
- [signup]: https://www.timescale.com/timescale-signup
- [self-hosted]: /how-to-guides/install-timescaledb/self-hosted/
\ No newline at end of file
+<!---
+Question for Jacob: How are the cards tooled for this page? --LKB 2021-09-22
```

Comment:
@jacobprall ☝️

Reply:
@ryanbooz and I met on this - we think it would be best to move install documentation up only one level, right below getting started, (all under TimescaleDB) for the time being. In the mean time, I will work on an improvement to the current install documentation architecture - see https://github.com/timescale/docs/issues/466 for more details. It will involve design and some decent engineering work, but I think the results will be worth it!

Reply:
OK. In that case, should we move Forge/Cloud Install docs under those headings then?

Reply:
I think the dream is to have the cards on the install page (including forge and cloud) reveal the install information in that page, similar to how the old docs worked

Reply:
Yeah, that's why I wanted to lift it out of the TimescaleDB section.

Reply:
@Loquacity I'll touch base with design today on a new icon